packages/react-components/review/beta/react-components.api.md

```diff
@@ -443,12 +443,14 @@ export interface DialpadButtonProps {
 // @beta
 export interface DialpadProps {
     // (undocumented)
-    onClickDialpadButton?: () => void;
+    onClickDialpadButton?: (buttonValue: string, buttonIndex: number) => void;
     // (undocumented)
     onDisplayDialpadInput?: (input: string) => string;
     // (undocumented)
     onSendDtmfTone?: (dtmfTone: DtmfTone) => Promise<void>;
     // (undocumented)
+    onTextFieldChange?: (input: string) => void;
```

Comment:
should this just be `onChange`
The end user doesn't have to know about the underlying implementation details.
Moreover, there will only be 1 `onChange` function on dialpad (As far as I can think)